devel/fatal/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        facebook fatal 2022.12.26.00 v
+revision            0
+categories          devel
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Library for fast prototyping software in modern C++
+long_description    Fatal is a library for fast prototyping software in modern C++. \
+                    It provides facilities to enhance the expressive power of C++. \
+                    The library is heavily based on template meta-programming, \
+                    while keeping the complexity under-the-hood.
+checksums           rmd160  cea1204d92930efee0da93c1444ebf53e60c6471 \
+                    sha256  40b7c4a644ad663232d255fffbdfd87be09872304165c67f46cc93c052319532 \
+                    size    651285
+github.tarball_from releases
+distname            ${name}-v${version}
+extract.mkdir       yes
+
+installs_libs       no
+supported_archs noarch
```

Comment:
can you please align this with the rest of the Portfile?

Reply:
I am sorry, not enough caffeine. Will fix all this now.